background_scripts/commands.js

```diff
@@ -386,11 +395,11 @@ const Commands = {
     "enterVisualLineMode",
     "toggleViewSource",
     "passNextKey",
-    "hardReload",
```

Comment:
Previously (last PR), hardReload was considered an advanced command, but reload is not.